frontend/src/lib/components/LemonSnack/LemonSnack.tsx

```diff
@@ -8,13 +8,22 @@ export interface LemonSnackProps {
     onClose?: () => void
     title?: string
     wrap?: boolean
+    className?: string
     'data-attr'?: string
+    color?: 'primary-extralight' | 'primary-highlight'
 }
 
-export function LemonSnack({ children, wrap, onClose, title }: LemonSnackProps): JSX.Element {
+export function LemonSnack({
+    children,
+    wrap,
+    onClose,
+    title,
+    className,
+    color = 'primary-highlight',
+}: LemonSnackProps): JSX.Element {
     return (
         <span
-            className={clsx('LemonSnack', {
+            className={clsx(`LemonSnack bg-${color}`, className, {
```

Comment:
I was told we were aiming to use utility classes as much as possible: https://posthog.slack.com/archives/C0113360FFV/p1668790081924109?thread_ts=1668789917.095769&cid=C0113360FFV

Reply:
Ah, of course, I didn't realize this is a utility class because my brain shortcircuited when I saw the two styles mixed in `.LemonSnack.bg-primary-highlight`. 😄